catalog-rest-service/src/main/resources/json/schema/entity/teams/team.json

```diff
@@ -76,6 +76,14 @@
       "$ref": "../../type/entityReference.json#/definitions/entityReferenceList",
       "default": null
     },
+    "childrenCount": {
+      "description" : "Total count of Children teams. An `Organization` can have `BusinessUnit`, `Division` or `Department` as children. A `BusinessUnit` can have `BusinessUnit`, `Division`, or `Department` as children. A `Division` can have `Division` or `Department` as children. A `Department` can have can have `Department` as children.",
+      "type": "integer"
```

Comment:
We can just leave this as childrenCount description. I think what each type of team can have as children is captured as part of children property. Is this count only immediate children or all the children?